tests/acceptance/01-InstallJoomlaCept.php

```diff
@@ -9,7 +9,7 @@
 // Load the Step Object Page
 $I = new \AcceptanceTester($scenario);
 
-$I->wantToTest('redCORE installation in Joomla 3');
+$I->wantToTest('Joomla 3 Installation');
```

Comment:
I have just updated the Comment, since we are just doing Joomla Installation in this test, I am not installing redCORE two times